doc/source/user/absolute_beginners.rst

```diff
@@ -89,6 +85,8 @@ you would enter. Everything that doesn't have ``>>>`` in front of it
 is **output**, or the results of running your code. This is the style
 you see when you run ``python`` on the command line, but if you're using IPython, you might see a different style.
 
+For more information, see `Documentation conventions
+<https://numpy.org/devdocs/doc_conventions.html>`_.
```

Comment:
Since this is an internal link, it should be something like (I think, must be verified that this actually works)
```suggestion
For more information, see :ref:`documentation-conventions`.
```

Reply:
Thanks, this slipped my mind.